app/routes/about-us.tsx

```diff
@@ -1,3 +1,83 @@
+import { Link } from '@remix-run/react';
+
 export default function AboutUs() {
-  return <h1 className="text-2xl my-5">The About Page For Social Plan-it</h1>;
+  return (
+    <>
+      <div className="bg-primary text-white flex flex-col justify-center content-center">
+        <div className="max-w-screen-md mx-auto sm:w-4/5 flex flex-col px-8 py-8 items-center justify-center">
+          <div className="w-[320px] flex self-start mx-auto pb-3">
+            <img
+              className="w-full"
+              src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1706807323/solar-system_pzkazh.png"
+              alt="Solar system"
+            />
+          </div>
+          <h2 className="w-50 pb-3 sm:w-100 font-bold sm:font-extrabold sm:text-4xl">
+            Ham hock pork shank, chislic chicken pork chop picanha. Alcatra turkey ribeye kevin fatback leberkas.
+          </h2>
+          <h3 className="w-50 sm:text-lg sm:w-90 sm:font-semibold pb-6">
+            Shank jerky shoulder short loin corned beef chislic ham cupim. Ribeye andouille jowl, short ribs chislic
+            leberkas ham rump sirloin jerky chicken turkey ground round.
+          </h3>
+          <div className="w-full sm:w-90 sm:items-end">
+            <Link
```

Comment:
I feel using Remix's Link component is unnecessary here as it shines for internal navigation. For external sources, an anchor tag would be enough. 🙂

Reply:
Link was removed and replaced with anchor tag